core/src/main/java/kafka/server/SharePartitionManager.java

```diff
@@ -116,18 +117,23 @@ public CompletableFuture<Map<TopicIdPartition, PartitionData>> fetchMessages(Fet
         return future;
     }
 
-    public CompletableFuture<ShareAcknowledgeResponseData> acknowledge(ShareSession session, PartitionInfo partitionInfo) {
+    public CompletableFuture<ShareAcknowledgeResponseData> acknowledge(TopicIdPartition topicIdPartition,
+                                                                       List<ShareAcknowledgeRequestData.AcknowledgementBatch> ackBatches) {
         throw new UnsupportedOperationException("Not implemented yet");
     }
 
     private SharePartitionKey sharePartitionKey(String groupId, TopicIdPartition topicIdPartition) {
         return new SharePartitionKey(groupId, topicIdPartition);
     }
 
-    // TODO: Implement session. Initial implementation shall create new session on each call which can
-    //  be used in further API calls.
-    public ShareSession session() {
-        throw new UnsupportedOperationException("Not implemented yet");
+    private SharePartition getSharePartition(SharePartitionKey sharePartitionKey) {
```

Comment:
In Kafka we do not use/avoid `get` prefix on method names hence `sharePartition`